Tests/Functional/Controller/ConnectTest.php

```diff
@@ -0,0 +1,204 @@
+<?php
+/**
+ * (c) shopware AG <info@shopware.com>
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Functional\Controller;
+
+use ShopwarePlugins\Connect\Tests\WebTestCaseTrait;
+use ShopwarePlugins\Connect\Tests\TestClient;
+use Symfony\Component\HttpFoundation\Response;
+
+class ConnectTest extends \PHPUnit_Framework_TestCase
```

Comment:
in generally this whole class looks very good and it is great to have such testCases
but wouldn't it be better to check not just the json response but also the database for request that change the db?
so we would have the backend tested end-to-end from api to the db